systemtest/src/main/java/io/strimzi/systemtest/templates/crd/KafkaTemplates.java

```diff
@@ -235,22 +247,6 @@ private static KafkaBuilder defaultKafka(Kafka kafka, String name, int kafkaRepl
                         .addToRequests("memory", new Quantity("256Mi"))
                         .build())
                 .endZookeeper()
-                .editEntityOperator()
```

Comment:
Should this part be kept? The settings for _shared memory_ might be different from the general settings.